src/main/scala/com/campudus/tableaux/arguments.scala

```diff
@@ -135,7 +142,8 @@ object ArgumentChecker {
     }
   }
 
-  def tryMap[A, B](tryFn: A => B, ex: => CustomException)(a: A): ArgumentCheck[B] = Try(OkArg(tryFn(a))).getOrElse(FailArg[B](ex))
+  def tryMap[A, B](tryFn: A => B, ex: => CustomException)(a: A): ArgumentCheck[B] =
+    Try(OkArg(tryFn(a))).getOrElse(FailArg[B](ex))
```

Comment:
It would be okay for me for one liners that it doesn't add a `{...}`-block around it. As soon as it gets more, I'd like to see where my method ends more easily.